spec.md

```diff
@@ -8,121 +8,160 @@ title: Semantic Specification
+                                       ↑
+                                       ↑
+                           (Span G "Follows" Span F)
 
 ~~~
 
 ~~~
-Temporal relationships for spans from a single trace
+Temporal relationships for all of the above
+
 
 ––|–––––––|–––––––|–––––––|–––––––|–––––––|–––––––|–––––––|–> time
 
  [Span A···················································]
    [Span B··············································]
       [Span D··········································]
     [Span C········································]
-         [Span E···········]   [Span F········]
-                                [Span G······]
-                                [Span H··]
-                                        [Span I·]
+         [Span E·······]        [Span F··] [Span G··] [Span H··]
 ~~~
 
 ## Traces
 
-A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system. A Trace can be thought of as a tree of Spans. (See the ASCII diagrams above)
+A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system. A Trace can be thought of as a directed acyclic graph (DAG) of Spans.
 
 ## Spans
 
-A **Span** represents a logical unit of work in the system that has a start time and a duration. Spans may be nested and ordered to model parent-child and causal relationships. Each span has an **operation name**, a presumably human-readable string which concisely names the work done by the span (e.g., an RPC method name, a function name, or the name of a subtask within a larger computation).
+A **Span** represents a logical unit of work in the system that has a start time and a duration. Spans may be nested and ordered to model causal relationships. Each span has an **operation name**, a presumably human-readable string which concisely represents the work done by the span (e.g., an RPC method name, a function name, or the name of a subtask or stage within a larger computation).
+
+<span id="references"></span>
+
+### Causal Span References
+
+A Span may reference zero or more Spans that are causally related. OpenTracing presently defines two types of references: `BelongsTo` and `Follows`. **Both reference types specifically model direct causal relationships between a descendant span and an ancestor span.** In the future, OpenTracing may also support reference types for spans with non-causal relationships (e.g., Spans that are batched together, Spans that are stuck in the same queue, etc).
+
+**`BelongsTo` references:** A descendant Span may "BelongTo" an ancestor Span. In a "BelongsTo" reference, the ancestor Span depends on the descendant Span in some fashion (often—but not always—the ancestor Span cannot finish until the descendant Span finishes). We sometimes refer to the "BelongsTo" ancestor as the "parent" and the descendant as the "child".
```

Comment:
If we are saying that it's turtles all the way down, then `ChildOf` is information-free; we can instead try and name the additional attributes of the relationship that provides the distinction. Practically speaking, the API for creating a child span does not have to require any role/relationship name in the happy case, which is caused-by/depends-on (or split/join, the way I prefer to think about it). 

Also, I notice that terms that focus on the _structure_ of the trace tend to miss the aspect of _flow_ that seems to underpin my thinking; the dynamic aspect of the execution we are tracing. Now that I think about it, that's probably the reason I liked `Follows` best.

Still thinking about something that would capture "split w/o join" or "no dependency". I agree that `Async` is not it.